source/reference/methods.txt

```diff
@@ -56,6 +56,54 @@ Administration Methods
      - Allows writes to continue on a database locked with
        :method:`db.fsyncLock()`.
 
+.. _mongosh-methods-atlas-search:
+
+Atlas Search Index Methods
+--------------------------
+
+:atlas:`Atlas Search indexes
+</atlas-search/atlas-search-overview/#fts-indexes>` let you query data
+in :atlas:`Atlas Search </atlas-search>`. Atlas Search indexes enable
+performant text search queries by mapping search terms to the documents
+that contain those terms.
+
+Use the following methods to manage Atlas Search indexes.
+
+.. important::
+
+   The following methods can only be run on deployments hosted on
+   :atlas:`MongoDB Atlas </>`.
+
+.. list-table::
+   :widths: 30 70
+   :header-rows: 1
+
+   * - Name
+
+     - Description
+
+   * - :upcoming:`db.collection.createSearchIndex()
```

Comment:
Note for review:

These methods don't have the same monospace styling as the other methods on the page. We can only use the `:method:` role for methods that are on the `/manual` branch of the server docs. Currently, the Atlas Search methods are only on the `:upcoming:` branch, so we have to use the `:upcoming:` role, which uses a slightly different font.

Once the Atlas Search methods are on the upcoming branch, we can use the `:method:` role here and these will look the same as the rest of the methods on the page.